migrations/migrate.go

```diff
@@ -0,0 +1,22 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+func Migrate(db *gorm.DB) {
+
+	m := gormigrate.New(db, gormigrate.DefaultOptions, []*gormigrate.Migration{
+		_202309271616_initial_migration,
+		_202309271617_fix_preimage_null,
+		_202309271618_add_payment_sum_index,
+	})
+
+	err := m.Migrate()
```

Comment:
I would just return err here and log the fatal error in main.go.
Package functions should never cause the program to exit.

Reply:
Good point. Thanks!